src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/RuntimeWrappedException.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 using System.Runtime.Serialization;
 
 namespace System.Runtime.CompilerServices
```

Comment:
Isn't `thrownObject` nullable?

Reply:
Yes, I'd expect both the `thrownObject` parameter and the `WrappedException` property to be nullable.

Reply:
Actually, I'm reconsidering this.  The only code that creates these exceptions should be the runtime, and it won't pass null.  Obviously with the ctor being public, anyone could in theory be constructing one of these, but no one should be.  And code that does get one of these can reasonably expect it the WrappedException to be non-null, since in any real code, it's only constructed with an object.

Given that, I'm inclined to say leave them both non-nullable.

Reply:
Yes that is why I didn't make them nullable, because this was only used by the runtime and I didn't expect the runtime to pass down a null `thrownObject`